### DISCUSSION TITLE
More badges to hunt

### DISCUSSION THREAD

**Discussion by annamedv** [ParanoidUser/yolo#1179] [Ask Away]
I see you like hunting badges 😉 I love it too!

Try to catch those badges as well: https://github.com/my-badges/my-badges

**Comment by ParanoidUser** [comment]
Ahoy, @annamedv!

Thank you for stopping by and sharing more shiny badges 🤩🎖️ The Delorean is my favorite so far (looking closely for the next flux capacitor birthday 👀). Great project! 👏👏

**Reply by annamedv** [reply]
DeLorean is my favourite too 😊

**Reply by lucas-0liveira** [reply]
nice badges, I'm interested
